src/Dumper/Mysql/TableFilterExtension.php

```diff
@@ -303,4 +303,18 @@ private function quoteValue(mixed $value): mixed
 
         return $value;
     }
+
+    /**
+     * Get a filter column so it can be safely injected in SQL query
+     * Extracts a possible expression to use instead of a real column name.
+     *
+     * @param Filter $filter Table filter
```

Comment:
Could you remove this annotation?
The type is already specified in the prototype of the method